app/src/DesignSystem.re

```diff
@@ -11,3 +15,44 @@ let margin = (top, right, bottom, left) =>
     marginBottom(px(bottom)),
     marginLeft(px(left)),
   ]);
+
+module Layout = {
+  let grid =
+    style([
+      display(grid),
+      gridTemplateColumns([`repeat((`num(8), `fr(1.0)))]),
+      gridTemplateRows([px(174), auto, px(96)]),
+      gridRowGap(px(0)),
+    ]);
+};
+
+module Positioning = {
+  let fullWidth = style([gridColumnStart(1), gridColumnEnd(9)]);
+  let wideWidthCentered = style([gridColumnStart(2), gridColumnEnd(8)]);
+  let mediumWidthCentered = style([gridColumnStart(3), gridColumnEnd(7)]);
+  let narrowWidthCentered = style([gridColumnStart(2), gridColumnEnd(8)]);
+};
+
+module Container = {
```

Comment:
As this is one of the most used component can we shorten it to something like `El`, it otherwise takes a lot of visual significance and distracts from the essential elements. I liked the [abstraction](https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/Element#el) in [elm-ui](https://github.com/mdgriffith/elm-ui).

Reply:
Good idea. I also converted the TwoColumn container to a bunch of Els, as it didn't feel like a good abstraction.